utils/mem/internal/cgroup/mem.go

```diff
@@ -63,11 +68,51 @@ func getMemStat(basePath, statName string) (int64, error) {
 // GetHierarchicalMemoryLimit returns hierarchical memory limit
 // https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt
 func GetHierarchicalMemoryLimit(basePath string) int64 {
+	return memStat(basePath, "hierarchical_memory_limit")
+}
+
+func getRSSMemory(basePath string) int64 {
+	return memStat(basePath, "total_rss")
+}
+
+func getWSSMemory(basePath string, used int64) int64 {
+	inactive := memStat(basePath, "total_inactive_file")
+	if used < inactive {
+		return 0
+	}
+	return used - inactive
+}
+
+func getWSSMemoryV2(basePath string, used int64) int64 {
```

Comment:
Is `v2` a clear enough name in this case? In the article they use something more "evocative" i.e. `cgroups.IsCgroup2UnifiedMode()`.